src/Serialization/Address.purs

```diff
@@ -128,15 +121,8 @@ instance Semigroup Slot where
 instance Monoid Slot where
   mempty = Slot zero
 
--- Needed for Haskell server. Slot is given by `getSlot` field.
-instance DecodeJson Slot where
-  decodeJson = jsonToAeson >>>
-    caseAesonObject
-      (Left $ TypeMismatch "Expected object")
-      (flip getField "getSlot" >=> Slot >>> pure)
-
-instance EncodeJson Slot where
-  encodeJson (Slot uint) = fromNumber (UInt.toNumber uint)
+instance EncodeAeson Slot where
+  encodeAeson' (Slot uint) = encodeAeson' (UInt.toNumber uint)
```

Comment:
Why did you remove decoding for Slot?

Reply:
I confirmed this with @vvtran , it's leftover code.